spec/v0-1.md

```diff
@@ -111,6 +120,18 @@ The expression `(eval <exp>)` or `(eval <exp> <env>)` first evaluates the expres
 
 While `eval` is part of core Lurk, it is not part of the initial `lurk-rs` release.
 
+## Numeral Parsing 
+
+The release specs of `lurk-rs` has diverged from core Lurk. The following parsing rules are only supported by `lurk-rs`. 
+
+The `lurk-rs` reader has support for parsing negative Nums `-<n>` and fractional Nums `<n>/<m>`. 
+
+A Num preceded by `-` is substracted from `0` at read time. 
+
+A Num `<n>` preceded by `/` and another Num `<m>` is calculated at read-time by performing a field-element division. This means that for arithmetic and algebraic purposes, the resulting field elements appear to behave like the corresponding rationals. However, note carefully, that this similarity does not extend to relational comparisons. For example, `(< 1/2 1/3) => t`, since `1/2` is the most negative Num, although this violates expectations if you consider 1/2 to behave like a rational.  
+
+Note: the parser for these has only one character of lookahead and hence are whitespace sensitive.
```

Comment:
Parser lookahead is an implementation detail that doesn't belong in a spec. However, it is true that the `-` and `/` notations are whitespace-sensitive — just like Common Lisp.

Reply:
Also, note that in order to implement these syntaxes, I had to use `peekmore::PeekMore` — exactly in order to obtain more than one-character lookahead in an edge case. :(